packages/playground/website-deployment/cors-proxy-config.php

```diff
@@ -86,12 +86,13 @@ public function obtain_token($remote_ip, $bucket_config) {
 			return false;
 		}
 
-		// @TODO: Handle IPv6 addresses in a way that cannot lead to storage exhaustion.
-		$ipv6_remote_ip = $remote_ip;
-		if (filter_var($remote_ip, FILTER_VALIDATE_IP, FILTER_FLAG_IPV4)) {
-			// Convert IPv4 to IPv6 mapped address for storage
-			$ipv6_remote_ip = '::ffff:' . $remote_ip;
-		}
+		/**
+		 * Aggregate addresses into /64 subnets.
+		 * This prevents a person with an entire /64 subnet from
+		 * exhausting the storage or getting more than their fair
+		 * share of the tokens.
+		 */
+		$remote_subnet = playground_ip_to_a_64_subnet($remote_ip);
```

Comment:
Hi @adamziel, I think we probably only want to do this after the number of rate-limited things in a subnet crosses some threshold. Let's add some conditional logic for this. If you're good with it, I should be able to make this change tomorrow.

Reply:
I guess this is a semi-theoretical change at the moment because we aren't going to see IPv6 addresses immediately but only after the hosting platform starts relaying IPv6 addresses. So defaulting to rate-limiting an entire block is maybe similar to what we will be doing when only receiving requests from IPv4 addresses.

Reply:
I went ahead and pushed a note about this. That should be good enough for now.